### PR TITLE
Pass config to external Kubemark cluster in e2e tests

### PR DESCRIPTION
**Pull request by bskiba**
When cluster autoscaler is used in kubemark tests,
pass default kubeconfig as external cluster config.

@shyamjvs @gmarek 

**Release note**:
```
NONE
```

### CODE REVIEW

File: test/kubemark/run-e2e-tests.sh [hunk 1]
```diff
@@ -32,6 +32,12 @@ export KUBE_MASTER_URL="https://${KUBE_MASTER_IP}"
 export KUBECONFIG="${ABSOLUTE_ROOT}/test/kubemark/resources/kubeconfig.kubemark"
 export E2E_MIN_STARTUP_PODS=0
 
+: ${KUBE_ENABLE_CLUSTER_AUTOSCALER:="false"}
```
Comment (shyamjvs) [inline test/kubemark/run-e2e-tests.sh]: Move this variable to `cluster/kubemark/gce/config-default.sh`. It's a useful option to be made visible.
Also rename it to sth like ENABLE_KUBEMARK_AUTOSCALER to not clash with the variable for the base cluster.

File: hack/ginkgo-e2e.sh [hunk 2]
```diff
@@ -155,4 +155,5 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   ${NUM_NODES:+"--num-nodes=${NUM_NODES}"} \
   ${E2E_REPORT_DIR:+"--report-dir=${E2E_REPORT_DIR}"} \
   ${E2E_REPORT_PREFIX:+"--report-prefix=${E2E_REPORT_PREFIX}"} \
+  ${KUBEMARK_EXTERNAL_KUBECONFIG:+"--kubemark-external-kubeconfig=${KUBEMARK_EXTERNAL_KUBECONFIG}"} \
```
Comment (shyamjvs) [inline hack/ginkgo-e2e.sh]: This is unnecessary. You can just pass `--kubemark-external-kubeconfig=${KUBEMARK_EXTERNAL_KUBECONFIG}` through `${@:-}` below.

File: test/kubemark/run-e2e-tests.sh [hunk 3]
```diff
@@ -32,6 +32,12 @@ export KUBE_MASTER_URL="https://${KUBE_MASTER_IP}"
...
+if [[ "${KUBE_ENABLE_CLUSTER_AUTOSCALER}" == "true" ]]; then
```
Comment (shyamjvs) [inline test/kubemark/run-e2e-tests.sh]: Move this if condition below and simply do `ARGS="${ARGS} --kubemark-external-config=${DEFAULT_KUBECONFIG}"`

File: hack/ginkgo-e2e.sh (hunk 2 above)
Comment (shyamjvs) [inline hack/ginkgo-e2e.sh]: Besides, changing lesser files => lesser approvals needed ;)

Comment (bskiba) [inline hack/ginkgo-e2e.sh]: Thanks for the suggestion! Fixed.

**Comment by shyamjvs** [comment]
Please squash the commits and then lgtm

**Pull request merged**